TerminalDocs/customize-settings/actions.md

```diff
@@ -698,7 +695,7 @@ _This command is not currently bound in the default settings_.
 > [!IMPORTANT]
 > This feature is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview).
 
-### Minimize to tray ([Preview](https://aka.ms/terminal-preview))
+### Minimize to tray
```

Comment:
I don't understand the direction of this change -- it looks like it's *deleting* "preview only" for tray

Reply:
Oh yep, looks like you're right. There was a miscommunication with the docs and these docs were added for last release, so I assumed they were coming out of preview for this one. I'll add the label back.